pyhive/hive.py

```diff
@@ -337,18 +346,14 @@ def fetch_logs(self):
 
 def _unwrap_column(col):
```

Comment:
Dropbox had some perf problems with the serialization stuff in the past. Downloading a bunch of rows from Hive ended up being Python CPU bound instead of network bound. I'll need to check on how this affects performance.